kubeflow/env.sh

```diff
@@ -5,6 +5,4 @@ export MGMT_NAME=<management-cluster-name>
 export MGMTCTXT=${MGMT_NAME}
 export LOCATION=<zone>
 
-export ASM_LABEL=asm-192-1
-
-
+source ./common/asm/env.sh
```

Comment:
We are already using ASM specific setup: For example, profile controller will remove the sidecar injection label and use ASM label. And the existing `kubeflow/env.sh` is defining ASM_LABEL env variable currently. 

So if users want to replace asm with istio, it requires many steps for them already, and this installation approach is not validated by us. (Istio/ASM is the foundation of many things: IAP, gateway, ingress, sidecar, etc.)

I want to make sure we only define the ASM label once in the whole directory, this is why I call the `common/asm/env.sh` script from root `env.sh`. And asking user to run `kpt cfg set -R asm-label ...` as single installation step by default seems to be redundant since we already ask them to run `bash kpt-set.sh` under `kubeflow/` directory. And keeping all shell commands in a bash script allows us to track it using git. Does it make sense to you?